src/test/java/core/basesyntax/model/GetByOperationTest.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.model;
+
+import static org.junit.Assert.fail;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class GetByOperationTest {
+    private static final String BALANCE = "b";
+    private static final String BALANCE_IN_UPPER_CASE = "B";
+    private static final String INCORRECT_OPERATION = "divide";
```

Comment:
Such entries should not be constants. You can store them in variables, or you can use them directly in a method if you only need a variable for it. Fix it everywhere